package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.82.0",
+  "version": "2.82.1",
```

Comment:
Nit: This should be a minor version bump (`2.83.0`) since it adds new functionality.